README.md

```diff
@@ -6,6 +6,9 @@
 
```

Comment:
- [ ] Please add a descriptive project `title and description`, the `Getting Started` section to give information on how to setup the project step by step.